### PR TITLE
cleanup manual value clamping: stores.cpp 2

### PR DESCRIPTION
**Pull request by bcko**
part of #1666

### CODE REVIEW

File: Source/stores.cpp [hunk 1]
```diff
@@ -2608,10 +2606,10 @@ void StorePrior()
 	PlaySFX(IS_TITLEMOV);
 	if (stextsel != -1 && stextscrl) {
 		if (stextsel == stextup) {
-			if (stextsval != 0)
+			if (stextsval != 0) {
 				stextsval -= 4;
-			if (stextsval < 0)
-				stextsval = 0;
+			}
+			stextsval = std::max(stextsval, 0);
```
Comment (ephphatha) [inline Source/stores.cpp:2612]:
```suggestion
			stextsval = std::max(stextsval - 4, 0);
```
The if != 0 statement is unnecessary, the decrement can be inlined and let std::max clamp to 0 in all cases.

Webkit Code Style guidelines forbid braces on single statement conditionals as well. (I prefer braces everywhere but try keep it consistent for this project 😅 )

Comment (bcko) [reply Source/stores.cpp]: Updated the pull request and looking into Webkit Code Style guideline (https://webkit.org/code-style-guidelines/). Thank you @ephphatha !

**Review by ephphatha — APPROVED** [review]

File: Source/stores.cpp (hunk 1 above)
Comment (AJenbo) [reply Source/stores.cpp]: > I prefer braces everywhere

Same :D

Comment (AJenbo) [reply Source/stores.cpp]: @bcko if you haven't see it we have some additional rules on top of what Webkit provides: https://github.com/diasurgical/devilutionX/wiki/Code-Style

**Pull request merged**